public/docs/ts/latest/guide/style-guide.jade

```diff
@@ -2061,7 +2059,11 @@ a(href="#toc") Back to top
 
 .s-why.s-why-last
   :marked
-    **Why?** Data service implementation may have very specific code to handle the data repository. This may include headers, how to talk to the data, or other services such as `Http`. Separating the logic into a data service encapsulates this logic in a single place hiding the implementation from the outside consumers (perhaps a component), also making it easier to change the implementation.
+    **Why?** Data service implementation may have very specific code to handle the data repository. 
```

Comment:
Replace #2062-2066

**Why?** The details of data management, such as headers, HTTP methods, caching, error handling, and retry logic, are irrelevant to components and other data consumers.

A data service encapsulates these details. It's easier to evolve these details inside the service without affecting its consumers. And it's easier to test the consumers with mock service implementations.